osu.Game/Graphics/UserInterface/SettingsDropdownContainer.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public abstract class SettingsDropdownContainer : Container
+    {
+        /// <summary>
+        /// The title of this container, which will be written in header.

```

Comment:
If class is renamed to ReplaySettingsGroup -> "The title to be displayed in the header of this group."